### PR TITLE
Mobile responsive Ticket Pricetable

### PR DESCRIPTION
**Pull request by 2root2**
Replaced `table` with responsive bootstrap-grid. 
Fix for Issue #138 
Refer [gh-pages](https://2root2.github.io/inpycon2018/)
@pythonindia/pycon-2018-team please review

### CODE REVIEW

**Comment by 2root2** [comment]
@vaibhavsingh97 It was not meant to be there in the first place, sorry my bad. Removed commented code.

File: custom/css/style.css [hunk 1]
```diff
@@ -771,31 +771,53 @@ padding-bottom:80px;
 .pycon2018_panel{
 
     width: 95%;
-padding:10px 1px;
+    /*padding:10px 1px;*/
```
Comment (vaibhavsingh97) [inline custom/css/style.css]: Please remove comment

File: custom/css/style.css [hunk 2]
```diff
@@ -771,31 +771,53 @@ padding-bottom:80px;
...
 }
 
+.pycon2018_panel .time{
+    font-weight:bold;
+    color:#e23e89;
+}
+
+.ticket-column {
+  position: relative;
+  height: 130px;
```
Comment (vaibhavsingh97) [inline custom/css/style.css]: use `em` instead of `px`. will make responsive according to the screen size

**Comment by vaibhavsingh97** [comment]
@2root2 Good job 🎉, Just fix Minor issues

File: custom/css/style.css (hunk 2 above)
Comment (2root2) [reply custom/css/style.css]: @vaibhavsingh97 
Tried using 10em for desktop, but looks too spacious for mobile, anything less then that messes with vertical alignment of buttons.
Attached screen-shot for 7em, vertical alignment of buttons still looks out of place. 
Do I have to use media query with `em` too? e.g `10em` on desktop and `5em` on mobile?
screen shot 2018-06-29 at 10 43 54 pm

value  |  desktop | Mobile
------ | --------- | --------
10em | 👍 | 👎 
5em | 👎  | 👍

**Comment by 2root2** [comment]
@vipulgupta2048 Shall I remove the space between heading and content? 
1. If we remove existing padding, watermarks to the right will overlap content (needs to be handled separately)
screen shot 2018-06-29 at 11 03 33 pm
screen shot 2018-06-29 at 11 02 56 pm

1. Remove just for mobile with media query

File: custom/css/style.css (hunk 2 above)
Comment (ananyo2012) [reply custom/css/style.css]: Ideally we should use `em` for everything to make everything responsive. But let's just use `px` in this case as it looks ok in both mobile and desktop views.

File: custom/css/style.css [hunk 3]
```diff
@@ -771,31 +771,53 @@ padding-bottom:80px;
...
     width: 95%;
-padding:10px 1px;
+    /*padding:10px 1px;*/
     margin: 0 auto;
 
     box-shadow: 0px 3px 10px 0px #ccc;
 
 }
 
-.pycon2018_panel .table>thead>tr>th{
```
Comment (ananyo2012) [inline custom/css/style.css]: Don't remove this css. Some other places are using this.

File: custom/css/style.css (hunk 1 above)
Comment (ananyo2012) [reply custom/css/style.css]: Yes remove this comment.

**Comment by ananyo2012** [comment]
@2root2 Can you remove the padding for heading and table just for the tickets section ?

**Comment by ananyo2012** [comment]
Overall this looks good. Just do the minor fixes and it can be merged.

**Comment by 2root2** [comment]
@ananyo2012 Made requested changes, Removed padding between Heading and content for ticket section (Note It also affect Speaker section since same class is used, However it looks good on mobile.) Have a look at [gh-pages](https://2root2.github.io/inpycon2018/)

**Comment by ananyo2012** [comment]
@vaibhavsingh97 Thanks for taking your time to review it. Really appreciate it.

**Pull request merged**